catalog/forms.py

```diff
@@ -0,0 +1,44 @@
+from django import forms
+from .models import Product
+
+
+class ProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        fields = ['name', 'description', 'price', 'category', 'image']
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Добавляем CSS-классы к каждому полю для стилизации
+        for field_name, field in self.fields.items():
+            field.widget.attrs.update({'class': 'form-control'})
+
+    def clean_name(self):
+        name = self.cleaned_data.get('name')
+        return self.check_forbidden_words(name)
+
+    def clean_description(self):
+        description = self.cleaned_data.get('description')
+        return self.check_forbidden_words(description)
+
+    def clean_price(self):
+        price = self.cleaned_data.get('price')
```

Comment:
Цена - это обязательное поле, оно не может отсутствовать
```suggestion
        price = self.cleaned_data['price']
```

Но вообще для цены нужен кастомный валидатор, например:
**forms.py**
```python
from django improt forms
from django.core.validators import MinValueValidator


class ProductForm(forms.ModelForm):
    price = forms.IntegerField(validators=[MinValueValidator(0)])

    class Meta:
        model = Product
        fields = ['name', 'description', 'price', 'category', 'image']
 
     ...
```